### PR TITLE
simulator: add some config to adjust different region size. 

### LINKED ISSUE #5259: add some config to adjust different region size.
The configuration of the region size should be changed, not hard code.

### PR DESCRIPTION
**Pull request by bufferflies**
### What problem does this PR solve?
new config is: 
```
sim-tick-interval="1s"
[raftstore]
capacity="6TB"
available="6TB"

[coprocessor]
region-split-size="512MB"
region-split-keys=1000000
```

Issue Number: Close #5259

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Manual test (add detailed scripts or steps below)

Code changes

Side effects

Related changes

### Release note

```release-note
None.
```

### CODE REVIEW

File: tools/pd-simulator/simulator/config.go [hunk 1]
```diff
@@ -41,20 +44,43 @@ const (
 	defaultLeaderPriorityCheckInterval = 100 * time.Millisecond
 )
 
+// unit of storage
```
Comment (lhy1024) [inline tools/pd-simulator/simulator/config.go:47]: https://github.com/tikv/pd/blob/34a4cce2cb07f8aa4283e28ba40a667e1321a1fb/tools/pd-simulator/simulator/cases/cases.go#L61 it is repeated

Comment (bufferflies) [reply tools/pd-simulator/simulator/config.go:47]: replace with unit.MiB .

**Comment by bufferflies** [comment]
@lhy1024  PTAL

**Comment by bufferflies** [comment]
PTAL  @lhy1024 again.

**Comment by bufferflies** [comment]
PTAL @CabinfeverB @lhy1024 again

**Review by lhy1024 — APPROVED** [review]

**Pull request merged**